src/utils/dates.ts

```diff
@@ -10,3 +10,13 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset: number) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function is24HoursLater(purchaseDate: Date): boolean {
+	const currentTime = new Date();
+
+	const timeElapsedInMilliseconds = Math.abs(
+		currentTime.getTime() - purchaseDate.getTime(),
+	);
+
+	return timeElapsedInMilliseconds >= ONE_DAY_IN_MILLISECONDS;
```

Comment:
I like that this is it's own function in it's own file.

Bonus: it's in a file that's already in a utils folder! So clean!